session_0/word_processing.rb

```diff
@@ -34,11 +50,37 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+  #raise NotImplementedError # TODO
+  if(word_1.length > word_2.length)
+    l = word_2.length
+  elsif(word_2.length >= word_1.length)
+    l = word_1.length
+  end
+  
+  max = 0
+  (l+1).times do |x|
```

Comment:
What are you iterating over in the loop?

Reply:
I'm iterating over the numbers from 1 to l+1, so that I can get the substring of word_1 and word_2. Is there any other approach to this?

Reply:
Yeah, read this: https://github.com/IRIS-NITK/IRIS-RoR-Bootcamp-2020/pull/13#discussion_r552376283